udata/mail.py

```diff
@@ -45,11 +52,11 @@ def send(subject, recipients, template_base, **kwargs):
     if not isinstance(recipients, (list, tuple)):
         recipients = [recipients]
 
-    debug = current_app.config.get('DEBUG', False)
-    send_mail = current_app.config.get('SEND_MAIL', not debug)
-    connection = send_mail and mail.connect or dummyconnection
+    tpl_path = kwargs.pop('security_tpl', f'mail/{template_base}')
```

Comment:
Two years ago Flask-security refactored their mail feature to remove dependancy to `Flask-Mail`. PR [here](https://github.com/Flask-Middleware/flask-security/commit/4d6e2d4ad16d62a45edf59c486910f6047be874e).
The new function changed needs another path to the mails template: `"security/email"` we had `"email/security"`.
It also now has a templating rendering [part](https://github.com/Flask-Middleware/flask-security/blob/4bac7bc2abf0bcd76f929bcf18561e195bfcf2c7/flask_security/utils.py#L690).
The modification here is to ensure only the rendered emails are the ones not related to security. 
The new function is [here](https://github.com/Flask-Middleware/flask-security/blob/4bac7bc2abf0bcd76f929bcf18561e195bfcf2c7/flask_security/utils.py#L671)